src/integrations/integrations.test.ts

```diff
@@ -80,6 +84,7 @@ export default (createApp: CreateAppFunc) => {
                   query: 'query test{ testString }',
               });
           return req.then((res) => {
+              destroyApp(app);
```

Comment:
Did `afterEach` not work? That would be preferable to adding the call explicitly in every test, I think.